sentry-spring/src/main/java/io/sentry/spring/webflux/SentryWebExceptionHandler.java

```diff
@@ -0,0 +1,45 @@
+package io.sentry.spring.webflux;
+
+import io.sentry.IHub;
+import io.sentry.SentryEvent;
+import io.sentry.SentryLevel;
+import io.sentry.exception.ExceptionMechanismException;
+import io.sentry.protocol.Mechanism;
+import io.sentry.util.Objects;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.core.annotation.Order;
+import org.springframework.web.server.ResponseStatusException;
+import org.springframework.web.server.ServerWebExchange;
+import org.springframework.web.server.WebExceptionHandler;
+import reactor.core.publisher.Mono;
+
+/** Handles unhandled exceptions in Spring WebFlux integration. */
+@Order(
+    -2) // the DefaultErrorWebExceptionHandler provided by Spring Boot for error handling is ordered
```

Comment:
Maybe use existing property for order tuning? `sentry.exception-resolver-order`. But, for some reason, it has default value 1 for MVC handler.